### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by fidlow**

### CODE REVIEW

**Review by a-chabin — APPROVED** [review]

**Comment by AtkishkinVlad** [comment]
**fix** very strange rendering icons in open state

Снимок экрана 2023-02-03 в 23 40 53

File: styles/globals.css [hunk 1]
```diff
@@ -10,7 +10,6 @@ body {
     max-width: 100%;
     overflow-x: hidden;
     font-size: 1rem;
-    scroll-behavior: smooth;
```
Comment (AtkishkinVlad) [inline styles/globals.css:13]: **question** why remove this property ?

Comment (a-chabin) [reply styles/globals.css:13]: It is duplicated and is above. This applies to all the rules that have been corrected

**Comment by a-chabin** [comment]
> **fix** strange rendering icons in open state
> 
> Снимок экрана 2023-02-03 в 23 40 53

Yep, strange. This will be fixed in #62

**Pull request merged**